index.html

```diff
@@ -18,274 +18,317 @@
 </head>
 
 <body>
-    <header>
-        <div id="statusbar">
-            <div>9:42</div>
-            <div id="status-icons">
-                <i class="fa-solid fa-signal"></i>
-                <i class="fa-solid fa-wifi"></i>
-                <i class="fa-solid fa-battery-full"></i>
+    <div class="container">
+        <header>
+            <div id="statusbar">
+                <div>9:42</div>
+                <div id="status-icons">
+                    <i class="fa-solid fa-signal"></i>
+                    <i class="fa-solid fa-wifi"></i>
+                    <i class="fa-solid fa-battery-full"></i>
+                </div>
             </div>
-        </div>
-        <div id="toolbar">
-            <div id="logo"><a href="#">My logo</a></div>
-            <nav>
-                <div id="menu"><a href="#"><i class="fa-solid fa-bars"></i></a></div>
-            </nav>
-        </div>
-    </header>
-
-    <section id="main-section">
-        <div class="column">
-            <span>
-                <div id="img-container"></div>
-            </span>
-            <h1>I'm Alexandra. Glad to see <br>you!</h1>
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-                work and<br> experience! If you like what you see and have a project you need coded, don’t hesitate to
-                contact me.</p>
-            <div id="social-media-icons">
-                <p>LET'S CONNECT</p>
-                <ul>
-                    <li> <a href="#">
-                            <i class="fa-brands fa-twitter"></i>
-                        </a></li>
-                    <li> <a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
-                    <li> <a href="#"><i class="fa-sharp fa-solid fa-m"></i></a></li>
-                    <li> <a href="#"><i class="fa-brands fa-github"></i></a></li>
-                    <li> <a href="#"><i class="fa-regular fa-hand-peace fa-flip-horizontal"></i></a></li>
-                </ul>
+            <div id="toolbar">
+                <div id="logo"><a href="#">My logo</a></div>
+                <nav>
+                    <div id="menu" class="show-mobile"><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+                    <div class="show-desktop">
+                        <ul>
+                            <li>Portfolio</li>
+                            <li>About</li>
+                            <li>Contact</li>
```

Comment:
- Awesome :wink: but I think you can make this more semantically correct to boost your search engine optimization, make you compliant with common best practices, and also increase your site's accessibility. Since this is intended to be a navigation list (as you have rightly used a `nav` tag), these are definitely the navigation items.
  - [ ] To make users interact with your page as expected, kindly wrap each of these in a `<li><a href="#id">Link here</a></li>` 

![image](https://user-images.githubusercontent.com/99715640/236652656-9fa378a6-5971-45fb-82aa-3f2ae88426c5.png)